modules/cpmso-config-options.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cpmso-config-prefix_{context}"]
+= Adding a custom prefix to control plane machine names
+
+You can customize the prefix of machine names created by a control plane machine set.
+This can be done by editing a control plane machine set custom resource (CR)
+
+.Procedure
+
+// Saw the step below in another procedure, let me know if it's good to keep it in here.
```

Comment:
Should we remove this?

Reply:
If you think the step below is valid to keep (which I am guessing is the case), then I'll go ahead and delete this comment.